themes/default/content/what-is/what-is-platform-engineering.md

```diff
@@ -77,8 +60,18 @@ Elkjøp Nordic is the leading consumer electronics retailer in the Nordics. The
+
+![Pulumi's solution for platform teams](/images/product/platform-teams-diagram.png)
+
+Pulumi's solution for platform teams encompasses all of the key requirements described earlier in this article:
 
-Choosing an infrastructure as code framework is an important foundation of a platform engineering strategy because it impacts how infrastructure is modeled, tested, distributed, and deployed across an organization. In particular, the choice of IaC language affects infrastructure’s usability, reusability, and capacity for scale. General-purpose languages provide major advantages because they are widely adopted already, have more flexibility and expressiveness than DSLs, and come with rich ecosystems of tools and frameworks that increase productivity. Last but not least, it’s much easier for developers to use IaC when it’s written in languages they already know.
+1. The _Developer Control Plane_ enables the simple and powerful user experience, allowing platform teams to meet the varying needs of different developers and development teams. The core infrastructure as code tool supports the languages that teams are already familiar with (including TypeScript, Python, Go, C#, Java, and YAML), and leverages the powerful ecosystems surrounding these languages (IDE support, linters, and test frameworks, for example). There's also support for internal developer portals like [AWS Proton](https://aws.amazon.com/proton/) and [Backstage](https://backstage.io).
+2. _Integration and Delivery_ is key to supporting automation as the default. This includes extensive integration with numerous CI/CD systems as well as Pulumi's own CI/CD offering, called [Pulumi Deployments](/product/pulumi-deployments/). The [Automation API](/automation/) makes it possible to embed IaC into application software, empowering teams to create reusable infrastructure workflows.
+3. _Monitoring and logging_ provides the "full visibility on everything" that is a requirement for platform engineering, through support for and integration with leading monitoring, logging, and metric solutions. [Pulumi Insights](/product/pulumi-insights/) provides advanced search (including natural language search), analytics, and artificial intelligence (AI) into [Pulumi Cloud](/product/pulumi-cloud/) to provide actionable knowledge on cloud usage and cost optimizations.
+4. _Security and identity_ ensure security is a foundational element. [CrossGuard](/crossguard/) provides policy-based controls---including remediation of policy violations---using the same general purpose programming languages that our core IaC offering supports. [Pulumi ESC](/product/esc) enables centralized access to secrets and configuration information.
```

Comment:
Want to make the 2nd word caps as well so it follows the other 2 bullets?

Reply:
I'll fix the other two bullets---our style guide calls for sentence case in situations like this. Thanks for pointing this out!